.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,21 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  build:
+    name: build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image_type: [pytorch]
+    env:
+      DOCKER_BUILDKIT: 1
+      IMAGE_TYPE: ${{ matrix.image_type }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - run: ./scripts/cibuild
+
+      - uses: codecov/codecov-action@v2
```

Comment:
Turns out Codecov [publishes an action](https://github.com/codecov/codecov-action), so we don't have to script this ourselves anymore. It works well!

We also don't need to maintain an upload token since this is a public repo.